quicklogic/common/utils/flatten_layout.py

```diff
@@ -0,0 +1,515 @@
+#!/usr/bin/env python3
```

Comment:
License header?

Reply:
It seems that we do not have any of those in SymbiFlow... Which one should I use? The same as in other SymbiFlow related repos? (like prjxray?)